mapit/templates/mapit/description.html

```diff
@@ -0,0 +1 @@
+Points-to-boundary mapping. Input a postcode or geographical location, and MapIt returns the relevant constituency or authority.
```

Comment:
Similarly, "Point to boundary" and "administrative area" here I think for the default.